dags/crossref_event_data_import_pipeline.py

```diff
@@ -207,7 +207,7 @@ def log_last_record_date(**kwargs):
 )
 crossref_event_data_etl_task = create_python_task(
     CROSSREF_DAG, "crossref_event_data_etl",
-    crossref_data_etl,
```

Comment:
might be slightly easier to read as a separate line (although it is the same above).

Just making a note that one can also set `default_args` on a DAG, that are applied to the tasks.
I am not sure whether it is an advantage or not. It might help keeping the DAG focused.

I used it in Data Science DAGs, as part of `create_dag`: https://github.com/elifesciences/data-science-dags/blob/455eb44b66f04b673bb97475dfa731d1e96a7963/dags/peerscout_recommend_senior_editors.py#L29-L31

Defined in: https://github.com/elifesciences/data-science-dags/blob/455eb44b66f04b673bb97475dfa731d1e96a7963/data_science_pipeline/utils/dags.py#L53-L89

Reply:
OK, I am separating them in lines.
Thank you for sharing `default_arg`, I will be helpful in case of a lot of tasks in a DAG I will use it then.